src/SonarQube.Client.RequestGenerator/Program.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading;
+using SonarQube.Client.Api.Requests;
+using SonarQube.Client.Services;
+
+namespace SonarQube.Client.RequestGenerator
+{
+    class Program
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a 'protected' constructor or the 'static' keyword to the class declaration. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=csharpsquid%3AS1118)